frontend/src/app/offers/offer-join-form/offer-join-form.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="container">
+  <form [formGroup]="joinForm" (ngSubmit)="onSubmit()">
+    <div class="form-group row">
+      <label class="col-form-label" for="{{ email }}">Email:</label>
+        <div class="col-md-5">
+          <input disabled class="form-control" formControlName="applicant_email" id="{{ email }}"/>
+        </div>
+      <label class="col-form-label" for="{{ fullname }}">Imię i nazwisko:</label>
+        <div class="col-md-5">
+          <input disabled class="form-control" formControlName="applicant_name" id="{{ fullname }}"/>
+        </div>
+    </div>
+    <div class="form-group row">
+      <label class="col-form-label" for="{{ phone }}">Telefon:</label>
+        <div class="col-md-5">
+          <input disabled class="form-control" formControlName="phone_no"/>
+        </div>
+    </div>
+    <div class="form-group-row">
+      <label class="col-md-3 col-form-label" for="{{ message }}">Możliwe uwagi:</label>
+      <div class="col-md-9">
+        <textarea class="form-control" rows="3" formControlName="message"></textarea>
+        <volontulo-form-error [fc]="joinForm.controls.message" [minLength]="10" [maxLength]="2000"></volontulo-form-error>
+      </div>
+    </div>
+    <div class="form-group row">
+      <div>
+        <button type="submit" class="btn-primary">
```

Comment:
![image](https://user-images.githubusercontent.com/28116755/45121986-ea437180-b162-11e8-861a-8e8426d98de2.png)
This form doesn't look good - smth is wrong with grid and RWD doesn't work as well.